python/nav/ipdevpoll/plugins/entity.py

```diff
@@ -51,7 +51,7 @@ def on_plugin_load(cls):
     @defer.inlineCallbacks
     def handle(self):
         self._logger.debug("Collecting physical entity data")
-        need_to_collect = yield self._need_to_collect()
+        need_to_collect = yield self._need_to_collect()  # noqa: F841 - needs work
```

Comment:
@lunkwill42 I think you can probably write a better comment here on why this needs work